src/app/pages/credentials/users/user-api-keys/components/api-key-form/api-key-form.component.html

```diff
@@ -32,14 +31,13 @@
           [tooltip]="tooltips.nonExpiring| translate"
         ></ix-checkbox>
 
-        <!-- TODO: https://ixsystems.atlassian.net/browse/NAS-132423 (Implement IxDatePickerComponent) -->
          @if (!form.controls.nonExpiring.value) {
-          <ix-input
-            formControlName="expiresAt"
-            [label]="'Expires at' | translate"
-            [tooltip]="tooltips.expires | translate"
-            [required]="true"
-          ></ix-input>
+            <ix-datepicker
```

Comment:
Let's use 

```
    relativeDateColumn({
      title: this.translate.instant('Expires Date'),
      propertyName: 'expires_at',
      getValue: (row) => row.expires_at?.$date || this.translate.instant('Never'),
    }),
```

For table, to see date in another format, like
<img width="226" alt="Screenshot 2024-11-25 at 14 05 18" src="https://github.com/user-attachments/assets/be14e90c-6a28-4292-9307-656ac2a4d526">